SA/design_concept/content/results/results_poc.typ

```diff
@@ -0,0 +1,21 @@
+= Proof of Concept
+The project produced a proof of concept application, that implements the design concept described in @design_proposal_iteration_2 and @design_specification.
+While the application doesn't implement all concepts required to learn functional programming, like the construction of custom data types, the application shows that the concept for function composition works and is easy to use.
```

Comment:
```suggestion
Although the application doesn't offer visualizations for all of the most common aspects of functional programming, such as the construction of custom data types, the application shows that the proposed concept for function composition works and is easy to use.
```

Although in both cases correct, I think it is a bit confusing to use the same word for two things in one sentence.